src/clr-angular/forms/combobox/option.spec.ts

```diff
@@ -20,7 +20,11 @@ import { OptionSelectionService } from './providers/option-selection.service';
             Test
         </clr-option>
     `,
-  providers: [IfOpenService, { provide: POPOVER_HOST_ANCHOR, useExisting: ElementRef }, OptionSelectionService],
+  providers: [
+    ClrPopoverToggleService,
+    { provide: POPOVER_HOST_ANCHOR, useExisting: ElementRef },
```

Comment:
Do we still need this? The intention with the `util/popover` directives and services is to remove all of the legacy popover code. I think this should be removed with this refactor.

Reply:
Similar, though not same, as the combobox.spec.ts stuff. 
Similar - it's also about code I'll very soon refactor with the new combobox code.
Not same - unlike combobox.spec.ts here it was easier for me to fix the "dead code" vs commenting it. It's too small.
I mean, in this commit, I'm only removing the IfOpenService and substituting it with ClrPopoverToggleService. But I am not yet pushing the new smart implementation.
So, you can see mixture of util/popover and popover/whatever code. Although combobox is still undead here, it does exist. I need to make the build pass. Here it was simple to get it fixed. In combobox.spec it was hard, so I just commented it to gain momentum. The value there is minimal compared to the gain. Here - it's equal - so I did it.